### PR TITLE
[SinMH] add some various templated websites

### PR DESCRIPTION
**Pull request by vinh2201**
Add some websites support SinMH template.

### CODE REVIEW

File: src/web/mjs/connectors/guoman8.mjs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import SinMH from './templates/SinMH.mjs';
+
+export default class guoman8 extends SinMH {
+
+    constructor() {
+        super();
+        super.id = 'guoman8';
+        super.label = 'guoman8';
```
Comment (27136773) [inline src/web/mjs/connectors/guoman8.mjs]: May add chinese name `国漫吧`

File: src/web/mjs/connectors/guoman8.mjs [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import SinMH from './templates/SinMH.mjs';
+
+export default class guoman8 extends SinMH {
```
Comment (27136773) [inline src/web/mjs/connectors/guoman8.mjs]: Prefer PascalCase for class and file names

File: src/web/mjs/connectors/iceKr.mjs [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import SinMH from './templates/SinMH.mjs';
+
+export default class iceKr extends SinMH {
```
Comment (27136773) [inline src/web/mjs/connectors/iceKr.mjs]: Prefer PascalCase for class and file names

**Review by 27136773 — APPROVED** [review]
Looks good

**Comment by vinh2201** [comment]
Updated. Please check.

**Pull request merged**